docs/introduction/overview.mdx

```diff
@@ -153,356 +20,114 @@ description: "🧙 A modern replacement for Airflow."
-  <Card
-    title="Documentation"
-    icon="book-open-cover"
-    href="/introduction/overview"
-  ></Card>
-  <Card title="Features" icon="wand-magic-sparkles" href="#features"></Card>
-  <Card
-    title="Core design principles"
-    icon="palette"
-    href="/design/core-design-principles"
-  ></Card>
-  <Card
-    title="Core abstractions"
-    icon="layer-group"
-    href="/design/core-abstractions"
-  ></Card>
-  <Card title="Contributing" icon="hand" href="/contributing/overview"></Card>
 </CardGroup>
 
----
-
-## 🏃‍♀️ Quick start
-
-You can install and run Mage using Docker (recommended), `pip`, or `conda`.
-
-### Install using Docker
-
-1. Create a new project and launch tool (change `[project_name]` to any name you want):
-
-```bash
-docker run -it -p 6789:6789 -v $(pwd):/home/src mageai/mageai \
-  /app/run_app.sh mage start [project_name]
-```
-
-  - If you want to run Mage locally on a different port, change the first port after `-p`
-  in the command above. For example, to change the port to `6790`, run:
-
-  ```bash
-  docker run -it -p 6790:6789 -v $(pwd):/home/src mageai/mageai \
-    /app/run_app.sh mage start [project_name]
-  ```
-
-2. Open http://localhost:6789 in your browser and build a pipeline.
+## 🦄 Make data magical
 
-- If you changed the Docker port for running Mage locally, go to the url
-`http://127.0.0.1:[port]` (e.g. http://127.0.0.1:6790) in your browser to
-view the pipelines dashboard.
+We put the _fun_ in fun-ctional programming. Mage is a hybrid framework that combines the best of both worlds: the flexibility of notebooks with the rigor of modular code.
 
-### Using `pip` or `conda`
+- **Extract** and synchronize data from 3rd party sources.
+- **Transform** data with real-time and batch pipelines using Python, SQL, and R.
+- **Load** data into your data warehouse or data lake using our pre-built connectors.
+- Run, monitor, and **orchestrate** thousands of pipelines without losing sleep.
 
-1. Install Mage
+<p align="center">
+<img src="https://github.com/mage-ai/assets/blob/main/mascots/mascots-shorter.jpeg?raw=true" noZoom />
+</p>
 
-```bash
-pip install mage-ai
-```
+### 1️⃣ Build
 
-or
+Have you met anyone who loves developing in Airflow? That's why we designed an easy developer experience that you'll enjoy.
 
-```bash
-conda install -c conda-forge mage-ai
-```
+- **Simple developer experience**: Start developing locally with a single command or launch a dev environment in your cloud using Terraform.
+- **Language of choice**: Write code in Python, SQL, or R in the same pipeline for _ultimate_ flexibility.
+- **Engineering best practices built-in**: Every step in your pipeline is a standalone file with modular code that's reusable and testable. No more DAGs with spaghetti code 🍝.
+- **Bring your favorite tools:** Write [dbt](dbt/overview) models, use your favorite [IDE](development/text-editor), [track changes with Git](getting-started/setting-up-git), and much much more.
```

Comment:
Could you use the absolute paths in the url instead of the relative paths?
The links don't work on staing.

Reply:
Some of the links are not broken links. They just redirect to the overview page. Are we able to catch them with `mintlify broken-links`?

Reply:
Fixed, this was a backslash issue